ta-endpoint.adef

```diff
@@ -0,0 +1,12 @@
+executables:
+{
+    taEndpoint = (endpointComponent)
```

Comment:
Why? `endpointComponent` sounds weird. Can you simply use `component(s)`?

Reply:
We will have other components inside directory. The enpointComponent just for endpoint's  component. Change to `components` will be misleading. How about change to `endpointLegato`?

Reply:
see also:https://github.com/legatoproject/legato-af/tree/master/apps/sample/mqtt

Reply:
No camelCase allowed for directory name.

Reply:
or simply use `legato`.